src/test/java/bio/terra/workspace/common/BaseConnectedTest.java

```diff
@@ -11,4 +11,7 @@
  */
 @Tag("connected")
 @ActiveProfiles("connected-test")
-public class BaseConnectedTest extends BaseTest {}
+public class BaseConnectedTest extends BaseTest {
+
+  public static final String bufferServiceDisabledEnvsRegEx = "alpha|staging|prod";
```

Comment:
Do we run tests in prod?

Reply:
No - I decided to put that there just in case